src/igvfd/schemas/analysis_set.json

```diff
@@ -92,6 +92,22 @@
                 "appscript": "https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0011-lipocyteprofiler/broad/images/Batch5/images/BR00101116/",
                 "igvf_utils": "https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0011-lipocyteprofiler/broad/images/Batch5/images/BR00101116/"
             }
+        },
+        "protocols": {
```

Comment:
When adding a calculated property, we don't want to add a non-calculated (i.e. submittable) property to the schema. This is actually clashing since now there are 2 properties with the same name. So the only change that needs to happen here is in the /types/file_set.py file.

Reply:
Removed the `protocols` property from `analysis_set.json` in schemas.